tests/test.ts

```diff
@@ -43,6 +43,31 @@ describe.skip('Test clasp create function', () => {
   });
 });
 
+// When running this test, please put in a script ID
```

Comment:
What do you mean "put in a script ID"?
Can we first create a test script ID, then clone it?

Reply:
Yeah, we can do one and hard code it for everyone, if you think that's better. Or use the one from the clasp create test.